Tests fail with Octave 5

An [auto-rebuild](https://apps.fedoraproject.org/koschei/build/6585992) with Octave 5 is now failing:
```
======================================================================
FAIL: test_completion (__main__.OctaveKernelTests) (text='one')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python3.7/site-packages/jupyter_kernel_test/__init__.py", line 135, in test_completion
    set(sample['matches']))
AssertionError: Items in the second set but not the first:
'onenormest'
----------------------------------------------------------------------
```
In Octave 5, `onenormest` was [removed](https://www.gnu.org/software/octave/NEWS-5.1.html#removed-functions-and-properties).

--- a/test_octave_kernel.py
+++ b/test_octave_kernel.py
@@ -19,8 +19,8 @@ class OctaveKernelTests(jkt.KernelTests):
 
     completion_samples = [
         {
-            'text': 'one',
-            'matches': {'ones', 'onenormest'},
+            'text': 'acos',
+            'matches': {'acos', 'acosd', 'acosh'},
         },
     ]
 

